config/initializers/markdown.rb

```diff
@@ -0,0 +1,25 @@
+require "rouge/plugins/redcarpet"
+
+module TemplateHandlers
```

Comment:
Can we add a test for this to check its converting generic Markdown into HTML, doing the appropriate code highlighting correctly and rendering specific markdown from the gov uk renderer? we should be able to do it as a view spec, something like:

```
describe TemplateHandlers::Markdown, type: :view do
  subject { rendered }

  context "when rendering generic markdown" do
    let(:markdown) do
      <<~MARKDOWN
        # Heading

        Lorem ipsum

        With a [link](https://www.gov.uk)

        With a https://www.gov.uk/autolink
      MARKDOWN
    end

    before do
      stub_template "test.md" => markdown
      render template: "test"
    end

    it { is_expected.to have_css("p", text: "Lorem") }
    it { is_expected.to have_css("h1", text: "Heading") }
    it { is_expected.to have_css("p", count: 3) }
    it { is_expected.to have_css('a[href="https://www.gov.uk"]', text: "link") }
    it { is_expected.to have_css('a[href="https://www.gov.uk/autolink"]', text: "https://www.gov.uk/autolink") }
  end

  context "when rendering GOV UK specific markdown" do
    ...
  end

  context "when rendering code snippets" do
    ...
  end
end
```